### ISSUE TITLE
tethys db configure not working on Windows when installed from new conda pacakge

### ISSUE THREAD

**Issue by swainn** [tethysplatform/tethys#454]
Running the `tethys db configure` command throws an error on Windows after installing from the new conda package:

```tethys db configure
[34mInitializing Postgresql database server in "C:\Users\nswain\.tethys\psql/data"...[0m
The files belonging to this database system will be owned by user "nswain".
This user must also own the server process.

The database cluster will be initialized with locale "English_United States.1252".
The default database encoding has accordingly been set to "WIN1252".
The default text search configuration will be set to "english".

Data page checksums are disabled.

creating directory C:/Users/nswain/.tethys/psql/data ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting default timezone ... US/Mountain
selecting dynamic shared memory implementation ... windows
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok

WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.

Success. You can now start the database server using:

    pg_ctl -D ^"C^:^\Users^\nswain^\^.tethys^\psql^/data^" -l logfile start

Starting Postgresql database server in "C:\Users\nswain\.tethys\psql/data" on port 5436...
waiting for server to start.... done
server started
Creating Tethys database user "tethys_default"...
CREATE ROLE
Creating Tethys database user "tethys_default"...
createdb: too many command-line arguments (first is "utf-8")
Try "createdb --help" for more information.
ERROR!!!
```

**Comment by swainn** [comment]
Running `tethys db status` indicates that the server is running:

```
pg_ctl: server is running (PID: 2460)
D:/miniconda3/envs/tethys/Library/bin/postgres.exe "-D" "C:/Users/nswain/.tethys/psql/data" "-p" "5436"
```

**Comment by swainn** [comment]
Fixed with #456
